.gitignore

```diff
@@ -7,7 +7,16 @@ local.properties
 .idea/
 *.iml
 
+# Eclipse
```

Comment:
As far as I know Eclipse doesn’t support it — at least yet. You can keep, not a big deal.